packages/main/src/Link.js

```diff
@@ -93,6 +93,10 @@ const metadata = {
 		wrap: {
 			type: Boolean,
 		},
+
+		_rel: {
```

Comment:
This doesn't need to be a property, because we don't expect anyone to pass it from the outside and we don't want this to change the state of the control and invalidate it. It can be completely calculated in the template context file and then passed to the template

Reply:
The control is the best place to cache a dummy anchor element. That's why we cannot move the calculation in LinkTemplateContext.